### ISSUE TITLE
Security Issue

### ISSUE THREAD

**Issue by KareemAllam** [MoussaElenany95/ECommerce-api#1]
Greeting, Team
I saw this awesome repository. Great job.

I noticed the ".env' file with the following text:
```
SECRET_KEY=**********
ClOUD_NAME=***********
CLOUD_KEY=688**********
CLOUD_SECRET=ykN6Y&************
```
This may be harmful to your hard work.
Once you can, remove the file from the repository and keep your work safe.

Best of luck to the whole awesome team.

**Comment by nahlaFathy** [comment]
yes sure we are still working  on it  .. thank you for your notice
